src/getters.ts

```diff
@@ -23,7 +24,7 @@ export function getOrCreateService(id: BigInt): Service {
   let service = Service.load(id.toString())
   if (!service) {
     service = new Service(id.toString())
-    service.status = 'Filled'
+    service.status = 'Confirmed'
```

Comment:
this is also a breaking change, need to share an official statement to marketplaces

Reply:
wo and just realised, it's Opened the default status

Reply:
Got mixed up in the function call order, that's why I mistakenly thought the other one was redundant.